src/main/java/pl/edu/agh/awi/persistence/model/Flight.java

```diff
@@ -0,0 +1,132 @@
+package pl.edu.agh.awi.persistence.model;
+
+import com.google.common.collect.Sets;
+import org.neo4j.graphdb.Direction;
+import org.springframework.data.neo4j.annotation.*;
+
+import java.util.Date;
+import java.util.Set;
+
+@NodeEntity
+public class Flight {
+
+    @GraphId
+    private Long id;
+
+    @Indexed(unique = true)
+    private Long flightId;
+    private FlightStatus status;
```

Comment:
I think status should be string. It currently can be ON_AIR or LANDED. I know for sure there is also SCHEDULED state. But as I found no specification for this field I cannot be sure there are no other possibilities.